docs/usage-as-end-user.md

```diff
@@ -125,36 +125,6 @@ If you don't want to configure anything for the `cloudControllerManager` simply
 
 ## WorkerConfig
 
-Multiple zones can be configured for a worker group of a GCP Shoot. The minimum number of machines in every worker group should be equal to or greater than the number of zones configured for that worker-group.
```

Comment:
Why is this documentation removed?

Reply:
With scale from zero enabled on GCP, we won't need to restrict user with `min machine in worker pool >= no of zones" , thats why I removed it.
The [PR](https://github.com/gardener/gardener-extension-provider-gcp/pull/296) which added this validation before had added the mentioned lines in the documentation , so I removed the same.

Reply:
Thanks! I had to read carefully what is actually removed as I was thinking you are removing a general documentation about how multiple availability zones to be configured. Thanks for clarifying ans sorry for the inconvenience